db/software-eol.db

```diff
@@ -16,6 +16,8 @@
 #
 # Alpine - https://alpinelinux.org/releases/
 #
+os:Alpine 3.15:2023-11-01:1169879680
```

Comment:
There seems to be an error in the converted time

Reply:
Fixed with my change